vsc_pdf_template_transformer/lib/models/tpl_repeater.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:json_annotation/json_annotation.dart';
+import 'package:pdf/widgets.dart';
+import 'package:vsc_pdf_template_transformer/utils/evaluator.dart';
+import 'package:vsc_pdf_template_transformer/utils/widget_json_converter.dart';
+import '../utils/widget_builder.dart' as wb;
+
+part 'tpl_repeater.g.dart';
+
+@JsonSerializable(
+  checked: true,
+  disallowUnrecognizedKeys: false,
+  explicitToJson: true,
+)
+class TplRepeater {
+  String className = 'TplRepeater';
+
+  @WidgetJsonConverter()
+  List<dynamic>? array;
```

Comment:
This might be - and probably will be - an expression string until we evaluate it - hence it won't be a list. This should just be `dynamic array`.